playbooks/templates/charts/argocd/argocd-values.yaml.j2

```diff
@@ -0,0 +1,65 @@
+# -- Provide a name in place of `argocd`
+nameOverride: argocd
+
+createAggregateRoles: false
+createClusterRoles: true
+
+crds:
+  install: true
+  keep: true
+
+## Server
+server:
+  extensions:
+    containerSecurityContext:
+      seccompProfile:
+        type: Unconfined
+
+  containerSecurityContext:
+    seccompProfile:
+      type: Unconfined
+
+  ingress:
+    enabled: true
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-prod
+      kubernetes.io/tls-acme: "true"
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    ingressClassName: "nginx"
+
+    hosts: 
+      - argocd.eco.tsi-dev.otc-service.com
+
+    paths:
+      - /
+
+    tls: 
+      - secretName: your-certificate-name
+        hosts:
+          - argocd.eco.tsi-dev.otc-service.com
+
+    https: true
+
+repoServer:
+  env:
+    - name: ARGOCD_GPG_ENABLED
+      value: "false"
+  containerSecurityContext:
+    seccompProfile:
+      type: Unconfined
+
+## ApplicationSet controller
+applicationSet:
+  containerSecurityContext:
+    seccompProfile:
+      type: Unconfined
+
+configs:
+  cm:
+    oidc.config: |
+      name: Keycloak
+      issuer: http://keycloak.eco.tsi-dev.otc-service.com/
+      clientID: argocd
+      clientSecret: $oidc.azuread.clientSecret
```

Comment:
that is pretty much wrong I guess. For reference look at how we get template values coming from secrets (i.e. for deploying prometheus)

Reply:
yes, it is fixed. 
but there is a problem how to create a secret.
https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/keycloak/#configuring-argocd-oidc